docs/en/engines/table-engines/integrations/s3.md

```diff
@@ -139,6 +139,7 @@ The following settings can be set before query execution or placed into configur
 - `s3_max_put_burst` — Max number of requests that can be issued simultaneously before hitting request per second limit. By default (`0` value) equals to `s3_max_put_rps`.
 - `s3_max_get_rps` — Maximum GET requests per second rate before throttling. Default value is `0` (unlimited).
 - `s3_max_get_burst` — Max number of requests that can be issued simultaneously before hitting request per second limit. By default (`0` value) equals to `s3_max_get_rps`.
+- `s3_max_inflight_parts_for_one_file` - Limits the number of put requests that can be run concurenly for one object. The value `0` means unlimited. Default value is `20`.
```

Comment:
Did we do any estimations/experiments? Maybe add a comment (in code) with motivation for `20`.

Reply:
Honestly, 20 is my guess. 
20 buffers with at least 5Mb is 100Mb inflight per file. If it 16Mb buffer it would be 320Mb. Quite large amount but it is better that infinity. Will add a comment.